Source/Core/Core/Config/MainSettings.cpp

```diff
@@ -44,6 +44,8 @@ const ConfigInfo<std::string> MAIN_GCI_FOLDER_B_PATH_OVERRIDE{
     {System::Main, "Core", "GCIFolderBPathOverride"}, ""};
 const ConfigInfo<bool> MAIN_GCI_FOLDER_CURRENT_GAME_ONLY{
     {System::Main, "Core", "GCIFolderCurrentGameOnly"}, false};
+const ConfigInfo<bool> GECKO_SYNC_OVERRIDE{
```

Comment:
This does not conform to naming the convention. Should be prefixed with `MAIN_`.